arc_evo.pro

```diff
@@ -4,6 +4,7 @@ HEADERS +=  \
   Stopwatch.hpp \
   environment.h \
   individual.h \
+  json.hpp \
```

Comment:
Mmm interesting, can yout tell me a bit more why this had to be declared here after the changes you have made? I do not think we need to do it in develop and I thought that the changes you had to implement here would not alter such design. It would be helpful for me if you can figure out what makes it not work.
Thanks in advance!

Reply:
Oh yeah, I added it when I was debugging and trying to figure out what was going wrong, but it is not actually needed - I thought adding it to QT would only change things in my personal environment, i didn't realize it was changing the code.